lom/src/pluginmgr/pluginmgr_common/pluginmgr_helper.go

```diff
@@ -325,14 +367,16 @@ func (plmgr *PluginManager) handleRequest(actionReq *lomipc.ActionRequestData) e
 
     handleResponseFunc := func(respData *lomipc.ActionResponseData) {
         if respData == nil {
-            lomcommon.LogPanic("In handleRequest(): Received nil response from plugin %v", actionReq.Action)
+            LogPanic("In handleRequest(): Received nil response from plugin %v", actionReq.Action)
+            return
         }
 
-        lomcommon.LogDebug("In handleRequest(): Received response from plugin %v", respData.Action)
+        LogDebug("In handleRequest(): Received response from plugin %v", respData.Action)
 
         if respData.Action != actionReq.Action {
-            lomcommon.LogPanic("In handleRequest(): Invalid action name received. Got  %v, expected %v",
+            LogPanic("In handleRequest(): Invalid action name received. Got  %v, expected %v",
                 respData.Action, actionReq.Action)
+            return
         }
 
         // For plugin with timeouts, stop the periodic log if its started due to previous request timeout
```

Comment:
long running plugins have heartbeats. Plugin's  error logic know when to send heartbeats or not.  If plugin logic determine that there id DB connection issues, it will stop sending heartbeats.

Reply:
resolved